src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/table.html.twig

```diff
@@ -51,7 +51,9 @@
         <tr>
           {% for column in grid.columns %}
             <td class="{{ column.type }}-type column-{{ column.id }}{% if column.options.clickable is defined and column.options.clickable %} clickable{% endif %}">
-              {{ column_content(record, column, grid) }}
+              <div {% if column.options.alignment is defined and column.options.alignment %} class="text-{{ column.options.alignment }}" {% endif %}>
```

Comment:
```suggestion
              <div class="text-{{ column.options.alignment }}">
```

Reply:
@Progi1984  some types redefine options, so always check if the option is defined. See for example https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Grid/Column/Type/Category/CategoryPositionColumn.php#L48